src/main/AutoLauncher.ts

```diff
@@ -17,35 +17,44 @@ export default class AutoLauncher {
         });
     }
 
-    isEnabled() {
-        return this.appLauncher.isEnabled();
+    async upgradeAutoLaunch() {
```

Comment:
This was in a different file for some reason, it uses the same library it just looks like there was 2 implementations for some reason. So I've removed that file and put the code here.